web/packages/extension/src/background.ts

```diff
@@ -193,7 +193,12 @@ async function enable() {
             },
             {
                 id: "4399",
-                matches: ["https://www.4399.com/flash/*"],
+                matches: [
+                    "*://www.4399.com/flash/*",
+                    "https://news.4399.com/qiu/",
+                    "https://my.4399.com/*",
+                    "http://sjsj.4399.com/",
```

Comment:
Is the presence/absence of a `*` at the end meaningful and intentional, or accidental? :thinking:

Reply:
The absence of a * is intentional because that is the game URL!😀

Reply:
Maybe group the general URLs with the wildcards and the game URLs together each? (basically switching lines 198 and 199)? But that's a minor nitpick; this generally looks good :)

Reply:
that's a nice nitpick!